File: VB/NorthwindDemo.Module/Module.vb
Imports System
Imports System.Collections.Generic
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace NorthwindDemo.Module
    Public NotInheritable Partial Class NorthwindDemoModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of DevExpress.ExpressApp.Updating.ModuleUpdater)
            Return ModuleUpdater.EmptyModuleUpdaters
        End Function
    End Class
End Namespace

File: CS/NorthwindDemo.Module/Module.cs
using System;
using System.Collections.Generic;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace NorthwindDemo.Module {
    public sealed partial class NorthwindDemoModule : ModuleBase {
        public NorthwindDemoModule() {
            InitializeComponent();
        }
        public override IEnumerable<DevExpress.ExpressApp.Updating.ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            return ModuleUpdater.EmptyModuleUpdaters;
        }
    }
}
